system/Debug/Toolbar/Collectors/Database.php

```diff
@@ -145,13 +145,22 @@ protected function formatTimelineData(): array
 			];
 		}
 
-		foreach (static::$queries as $query)
+		if (count(static::$queries) > 0)
```

Comment:
use truthy check instead (faster):

```suggestion
              if (static::$queries)
```